src/components/TransactionBlock/PlrDaoStakingTransactionBlock.tsx

```diff
@@ -517,11 +537,41 @@ const PlrDaoStakingTransactionBlock = ({
         />
         <NetworkAssetSelectInput
           label="To"
-          selectedNetwork={supportedChains[1]}
+          selectedNetwork={filteredSupportedChains[1]}
```

Comment:
what does it mean `1`? array order can change so make sure you select by chain ID and add a comment why this chain ID is selected